### DISCUSSION TITLE
Block & Unblock devices

### DISCUSSION THREAD

**Discussion by suxus-zz** [Art-of-WiFi/UniFi-API-client#94] [Q&A]
Hello, I installed the UniFi API client on my Synology web server (test_connection.php, works). Now i want to block and unblock devices and I saw in the examples that there is a block_list.php and an unblock_list.php. What is not clear to me, however, is how I can block a device now

http: //my-server/unifi/block_list.php

I should still be able to transfer the MAC address as a parameter somehow, or do I have to enter the MAC address in the file myself?

I hope someone here can help me with my first steps.
Many Thanks

**Comment by malle-pietje** [comment]
@suxus-zz This comes down to basic PHP programming skills I'm afraid. I suggest you reply to the thread below on the Ubiquiti community, maybe someone there can help you out?
https://community.ui.com/questions/PHP-client-class-to-access-the-UniFi-controller-API-updates-and-discussion/86cff6e2-06ad-46a2-8e0d-d91004f78752
